tests/lib/apt.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+apt_install_local_packages() {
+    if [ "$SPREAD_SYSTEM" = "ubuntu-14.04-64" ]; then
+        # relying on dpkg as apt(-get) does not support installation from local files in trusty.
+        dpkg -i "$@" || apt-get -f install -y
```

Comment:
Again, let's please drop the || here. This logic will ignore real problems. If the purpose is to ignore dependencies, let's use the respective flag of dpkg, and call apt-get on the next line.